packages/payment-widget/src/lib/components/payment-complete.svelte

```diff
@@ -1,5 +1,31 @@
 <script lang="ts">
   import { fade } from "svelte/transition";
+  import { exportToPDF } from "@requestnetwork/shared-utils/generateInvoice";
+  import { getCurrencyFromManager } from "@requestnetwork/shared-utils/getCurrency";
+  import { initializeCurrencyManager } from "@requestnetwork/shared-utils/initCurrencyManager";
+
+  export let createdRequest: any;
+  export let enablePdfReceipt: boolean = true;
+  export let enableRequestScanLink: boolean = true;
+  export let sellerLogo: string = "";
+
+  async function handleDownloadReceipt() {
+    if (createdRequest) {
+      try {
+        const currencyManager = initializeCurrencyManager([]);
+
+        const currencyData = createdRequest?.inMemoryInfo?.requestData;
+
+        await exportToPDF(
+          currencyData,
+          getCurrencyFromManager(currencyData.currencyInfo, currencyManager),
+          sellerLogo
+        );
+      } catch (error) {
+        console.error("Error downloading receipt:", error);
```

Comment:
I think we need a sonner or alert when this error occurs so the user is informed.